src/PhpSpreadsheet/Calculation/Statistical.php

```diff
@@ -3,6 +3,8 @@
 namespace PhpOffice\PhpSpreadsheet\Calculation;
 
 /* LOG_GAMMA_X_MAX_VALUE */
+use PhpOffice\PhpSpreadsheet\Calculation;
+
 define('LOG_GAMMA_X_MAX_VALUE', 2.55e305);
```

Comment:
namespace importing is between a comment and define. Not sure if this constant should be there (maybe refactor to a class constant)